app/scripts/modules/core/authentication/userMenu/userMenu.directive.js

```diff
@@ -18,8 +18,9 @@ module.exports = angular.module('spinnaker.core.authentication.userMenu.directiv
       link: function(scope) {
         scope.authEnabled = settings.authEnabled;
         scope.user = authenticationService.getAuthenticatedUser();
-        scope.showLogOutDropdown = () => authenticationService.getAuthenticatedUser().authenticated;
+        scope.showLoggedUserMenuDropdown = () => authenticationService.getAuthenticatedUser().authenticated;
```

Comment:
Probably could come up with a better name here.  What do you think about just `showUserMenu`?